language/stdlib/modules/validator_config_2.move

```diff
@@ -105,4 +134,52 @@ module ValidatorConfig2 {
     public fun rotate_consensus_pubkey_of_sender(new_consensus_pubkey: vector<u8>, proof: vector<u8>) acquires T {
         rotate_consensus_pubkey(Transaction::sender(), new_consensus_pubkey, proof);
     }
+
+    // TODO(philiphayes): add necessary rotation methods for discovery_config
+
+    // Public accessor for validator's network_identity_pubkey
+    public fun get_validator_network_identity_pubkey(config_ref: &Config): vector<u8> {
+        *&config_ref.network_identity_pubkey
```

Comment:
this seems unnecessary, we should just directly access this field, no?

Reply:
I don't think we can access field outside of module's code...